srcpkgs/mesa/patches/add-use-elf-tls.patch

```diff
@@ -1,28 +0,0 @@
-diff --git meson.build meson.build
-index 4afd8ca..ab9b365 100644
---- meson.build
-+++ meson.build
-@@ -378,7 +378,8 @@ if with_egl and not (with_platform_drm or with_platform_surfaceless or with_plat
- endif
- 
- # Android uses emutls for versions <= P/28. For USE_ELF_TLS we need ELF TLS.
--if not with_platform_android or get_option('platform-sdk-version') >= 29
-+with_use_elf_tls = get_option('use-elf-tls')
-+if with_use_elf_tls and not with_platform_android or get_option('platform-sdk-version') >= 29
```

Comment:
Please don't remove this patch, musl will break (issue https://gitlab.freedesktop.org/mesa/mesa/issues/966). This line needs to be adapted to include the changes (they added windows to the non-USE_ELF_TLS platforms).